app/controllers/int_api/marketplaces_controller.rb

```diff
@@ -49,6 +49,8 @@ def create
     auth_token = UserService::API::AuthTokens.create_login_token(user[:id])
     url = URLUtils.append_query_param(marketplace[:url], "auth", auth_token[:token])
 
+    ab_test_onboarding(community_id: marketplace[:id], alternative_flags: [:onboarding_new, :onboarding_old])
```

Comment:
What if we would just use one flag, `:onboarding`? Marketplaces that have this flag turned on would have the new onboarding flow and the once that don't have this would have the old (=current) getting started page.